features/git-hack/folder_does_not_exist/branch_exists_locally.feature

```diff
@@ -0,0 +1,38 @@
+Feature: git hack: starting a new feature from the main branch (with remote repo)
+
+  As a developer working on a new feature on the main branch
+  I want to be able to create a new up-to-date feature branch and continue my work there
+  So that my work can exist on its own branch, code reviews remain effective, and my team productive.
```

Comment:
This should be updated to something like https://github.com/Originate/git-town/blob/master/features/git-sync/folder_does_not_exist_on_main_branch/no_conflict.feature#L3-L5